### DISCUSSION TITLE
rMAPI is being archived

### DISCUSSION THREAD

**Discussion by juruen** [juruen/rmapi#313] [Announcements]
👋  Hi all,

I haven't had time to work on `rMAPI` for a while now.  The project is effectively unmaintained at this point. 

The reMarkable peeps continue to improve their product and that involves changes that break `rMAPI` every now and then. As I don't have time for working on it,  I'll be archiving the project in the next few weeks.

`rMAPI` was my first Go project ever,  and you can really see that in the code! It was born to provide a CLI  to deal with your  cloud reMarkable files like you would do with an FTP client -yup, I'm that old. With that in place I could  finally scratch an itch of mine: building  a  "print to reMarkable" functionality to use the reMarkable as a printer in just one click. It was great and worked as expected. Some other people found it also useful and we even had a few contributors that did an amazing job  expanding its functionality even further, some of them  became the actual project maintainers for a while.

The reality is that my life got busier, and wasn't using  my reMarkable so often any more. This meant I didn't have time to work on this project either. This has been the case for a long time now, and that's why I think it's reached a point where archiving `rMAPI` is the best path forward to reflect its current state.

It's been definitely a fun ride, but most of all I want to thank you everyone for your help with `rMAPI`!

**Comment by penguineer** [comment]
Thank you for your effort and good luck!

**Comment by Malanius** [comment]
Thank you for your amazing work so far!

It's a little bit sad that `rMAPI` to see the end of this era. It's been the best tool for managing files on my reMarkable. Never found anything that could match it - well, never had to :wink: If anyone knows possible alternatives, let me know, but I'll be using this until it breaks completely.

With that said it would be great if someone could carry the torch, if that is an option.

**Comment by anoduck** [comment]
@juruen I sincerely wish you the best in your journey through life. I have enjoyed using the project for a number of years, and it holds a warm place in my memory. Thanks for those years.

**Comment by mu1tiplex** [comment]
Too bad - thank you for all the work - it saved my life.
(well, hyperbole, but close)
